### DISCUSSION TITLE
Single cell label-free DIA quantification

### DISCUSSION THREAD

**Discussion by RalfG** [Proteobench/ProteoBench#426] [report concern about a live module]
### Aim of the new module

This proposed module addresses a need for benchmarking identification and quantification workflows for low-input and single cell proteomics (SCP). Currently, it is not known whether benchmarking results from high-input samples can be transferred to low-input proteomics. Indeed, typical low-input / single-cell datasets have different properties from bulk proteomics datasets, for instance shorter gradients, larger impact of contaminants, etc.

### Full description of the new module

Many acquisition and labeling strategies have been proposed and tested for SCP. Currently, the field seems to mostly converge towards label-free data independent acquisition (DIA) as SCP method of choice. Compared to various labeling strategies, label-free proteomics is highly accessible and simplifies sample preparation. Compared to data-dependent acquisition, DIA promises to lead to fewer missing values and a more consistent quantification accuracy.

We therefore propose to use a label-free DIA dataset as a first SCP benchmarking module. Similarly to the existing bulk proteomics quantification benchmarking modules, multiple low-input multispecies spike-in datasets are available that approximate a ground truth for assessing quantification accuracy. Using a similar type of dataset allows us to reuse much of the codebase and the metrics of the existing quantification ProteoBench modules.

Recently, a two proteome mix of HeLa and yeast acquired on a Thermo Orbitrap Astral was submitted as a preprint on bioRxiv (https://doi.org/10.1101/2024.02.01.578358). The dataset contains data for low-input samples of 250 pg in three different ratios of the two species. We propose to use the 240:10 and 200:50 ratios in the benchmark, as these are the closest to each other, reflecting a scenario closer to real-life differential protein abundances. 

Starting from the code base of the existing DIA quantification module, only limited changes will be required. For instance, only two species are present in this dataset, instead of three. Secondary metrics that are more specific to low-input proteomics could also be added, such as the missingness levels.

For this module, various datasets were considered. For instance, a technical HeLa dataset spiked in with E. coli proteins was recently acquired on a timsTOF Pro 2 instrument (PXD053462, https://doi.org/10.1038/s41467-024-52605-x). However, we opted to not include this dataset for benchmarking yet, as the newer timsTOF Ultra 2, or the timsTOF SCP, are more commonly used for low-input proteomics. Interestingly, this dataset contains specific higher-input runs to be used as matching enhancer (i.e., a run that can be used for matching-between-runs, optimizing the search engine’s internal spectral library of the experiment). As this method is steadily increasing in popularity, we propose to add a specific module to benchmark such approaches in the near future.

SCP is a young and fast-moving field. We expect that in the near future, additional modules for other labeling and acquisition setups could be added to ProteoBench. Notably, there is a high interest in novel DIA-compatible multiplexing strategies that allow for an increased sample throughput.

Unfortunately, low-input spike-in samples still do not accurately reflect real-world single cells, but are currently the only way to obtain an approximation of ground truth for quantification. If alternative benchmarking methods with actual single cell data becomes available, a new module should be added with such a dataset.

### Potential reviewers
Laurent Gatto (@lgatto)

### Will you be able to work on the implementation (coding) yourself, with additional help from the ProteoBench maintainers?

- [X] yes
- [ ] no

### Any other information

This proposal was conceived at the pre-conference hackathon session of the [iSCMS 2024](https://singlecellms.org/) conference at DTU, Denmark by the following participants:

- Benjamin Furtwängler (@bfurtwa)
- Kevin Yang (@yangkl96)
- Sam van Puyenbroeck (@SamvPy)
- Ralf Gabriels (@RalfG)

**Comment by wolski** [comment]
Hi,

For the single-cell dataset, the bias (estimated mean per species vs expected) is quite large. Thought, first it is a bug, somewhere in the Python code. However, according to the toml, the expected fold changes are 0.263 for human and -2.322 for yeast. For all three tools, the log2FC estimates are approximately 0 (Human) and -2 (Ecoli).


<img width="2286" height="1198" alt="image" src="https://github.com/user-attachments/assets/d6fac7a6-c490-439f-bfe4-6eae417aae5e" />


A possible explanation of why we observe for the Single Cell module log2FC_H ~ 0 and log2FC_Y ~ 2 is:

DIA-NN normalization
- **Global normalization:** Select low-CV precursors, sum their intensities, scale all runs to equal totals.
- **RT-dependent normalization:** Running median of fold changes across retention-time bins — different correction factors for different RT regions.
**Setup:**
Sample A: Human 200 pg, Yeast 50 pg
Sample B: Human 240 pg, Yeast 10 pg
Human CV: low (~1.2× change)
Yeast CV: high (~5× change)
**DIA-NN normalization steps:**
1. Order by CV — Human precursors rank low, Yeast rank high.
2. Select low-CV precursors — almost exclusively Human.
3. Sum intensities of selected precursors:
   - A: ~200 (Human-dominated)
   - B: ~240 (Human-dominated)
4. Scale to equalize totals: factor for B = 200/240 = 0.833
5. After scaling:
   - Human B: 240 × 0.833 = 200 → log2FC = 0 ✓
   - Yeast B: 10 × 0.833 = 8.33 → log2FC = log2(50/8.33) = +2.58 ✓
Therefore, Human log2FC ≈ 0 is expected.
The question is: why ~2.0 for Yeast?

It is very likely ratio compression. I observed this in other modules where the expected log2 fold change for *E. coli* is 2.0. The mean fold change over all ions for *E. coli* is smaller than two across all software.

<img width="720" height="397" alt="image" src="https://github.com/user-attachments/assets/eddaa034-d133-4fa7-b78d-3fc6ae217b92" />

It appears as if the software is reluctant to report log2FC > 2 (see, for example, Spectronaut SC data).

<img width="720" height="675" alt="image" src="https://github.com/user-attachments/assets/56536f66-b5cd-4ce9-b5a5-553d32f81046" />

**Reply by SamvPy** [reply]
Hi Wittold,

After discussing with the creators of this dataset, it seemed that the problem was related to the dataset (known deviation from expected ratio's), not necessarily the search engines. Therefore, we choose to archive this dataset and instead propose two new low-input (200pg) modules (Astral and timsTOF Ultra 2) from a different dataset. This is implemented in PR #1144
